doc/GOVERNANCE.md

```diff
@@ -0,0 +1,96 @@
+
+As mentioned in the prior section, Matz can overrule this document. That also means that Matz can direct existing Ruby core members who are on the `access` team to give another developer the ability to commit to the `ruby/rubygems` repository. We ask that any `access` team members follow the suggestions documented in the `access` section below. Specifically by notifying other `access` team members and documenting the change in an appropriate location. We encourage Matz to use this power sparingly, but ultimately place no demands on him and will respect his discretion in using it as he sees fit.
+
+Core (`core`) members of `ruby/rubygems` are not automatically added to Ruby core contributors. That process is described here [https://github.com/ruby/ruby/wiki/Committer-How-To#how-to-register-you-as-a-committer](https://github.com/ruby/ruby/wiki/Committer-How-To#how-to-register-you-as-a-committer).
+
+----
+
+## The `ruby/rubygems` Repository Provisional Governance
+
+When not explicitly directed by Matz, the rest of this document defines the governance expectations of [github.com/ruby/rubygems](http://github.com/ruby/rubygems) participants.
+
+## Provisional status
+
+This document is provisional. It is meant to be replaced so that governance stays relevant and applicable. Those involved in the `ruby/rubygems`  repository governance are encouraged to start drafting a provisional replacement governance model immediately and iterate on it as soon as possible.
+
+## Teams and Membership
+
+Each team, except where explicitly stated as being under the guidance of another team, will be self-managed: a current member proposes a new member, a majority of current members approve the selection, and this outcome (of who is currently on the team) is documented publicly. Any member may voluntarily resign at any time. For non-voluntary removal, the same consensus process is followed.
+
+Team membership is expected to be an active status. Those not actively exercising their status must be placed into a “paused” or “alumni” state (depending on the duration and discretion of the team). The access team should be notified of any changes in team status. Individual teams will determine a process for any “paused” or “alumni” members wishing to regain access.
+
+Team memberships are not mutually exclusive.
+
+### Teams
+
+* Access - controls access mechanism and governance
+* Release - controls software releases
+* Security - controls security matters
+* Core - controls software roadmap
+* Committer - controls contributions to the software
+* Triage - controls triaging of software issues
+
+## Access Team (`access`)
+
+The access team controls the mechanisms by which other teams have capabilities. i.e. they can give someone commit permissions on GitHub or owner/deploy permissions on RubyGems.org.
```

Comment:
I'm not sure about including rubygems.org as an example.  So far, we're not including rubygems.org in this governance doc so someone might interpret this example as something within scope.